src/main/java/org/generationcp/middleware/dao/dms/ProjectPropertyDao.java

```diff
@@ -180,6 +180,27 @@ public int getNextRank(final int projectId) {
 		}
 	}
 
+	public Map<Integer, List<Integer>> getEnvironmentVariablesByStudyId(final List<Integer> studyIds) {
```

Comment:
Should we filter by VariableType.ENVIRONMENT_DETAIL and VariableType.ENVIRONMENT_CONDITION?

Reply:
Or if not filtering by variable type, we should rename as `getEnvironmentDatasetVariables(final List<Integer> studyIds)`